src/amazons3/Module.md

```diff
@@ -2,12 +2,13 @@ Connects to Amazon S3 from Ballerina.
 
 # Module Overview
 
-The Amazon S3 client allows you to access the Amazon S3 REST API through ballerina. The following section provide you the details on connector operations.
-
+The Amazon S3 client allows you to access Amazon S3 REST API through ballerina. The following section provide you the 
```

Comment:
```suggestion
The Amazon S3 client allows you to access Amazon S3 REST API through Ballerina. The following section provides  
```

Better if we change "ballerina" to "Ballerina" in all instances whenever we see it.

Reply:
Fixed